.azure-pipelines-templates/daily-matrix.yml

```diff
@@ -7,15 +7,30 @@ parameters:
   build:
     NoSGX:
       cmake_args: '-DCOMPILE_TARGETS=virtual'
-    daily:
+    SGX:
+      cmake_args: '-DCOMPILE_TARGETS=sgx'
+    debug:
       cmake_args: '-DCMAKE_BUILD_TYPE=Debug -DSAN=ON -DCOVERAGE=ON -DBUILD_SMALLBANK=OFF'
+    install:
+      install_prefix: '/tmp/ccf-install'
+      cmake_args: '-DCMAKE_INSTALL_PREFIX=/tmp/ccf-install'
 
 jobs:
   - template: common.yml
     parameters:
       target: NoSGX
       env: '${{ parameters.env.NoSGX }}'
-      cmake_args: '${{ parameters.build.daily.cmake_args }} ${{ parameters.build.NoSGX.cmake_args }}'
+      cmake_args: '${{ parameters.build.debug.cmake_args }} ${{ parameters.build.NoSGX.cmake_args }}'
       suffix: 'Coverage'
       artifact_name: 'NoSGX_Coverage'
-      ctest_filter: ''
\ No newline at end of file
+      ctest_filter: ''
+
+jobs:
+  - template: common.yml
+    parameters:
+      target: SGX
+      env: '${{ parameters.env.SGX }}'
+      cmake_args: '${{ parameters.build.install.cmake_args }} ${{ parameters.build.SGX.cmake_args }}'
+      suffix: 'Release'
+      artifact_name: 'SGX_Release'
+      ctest_filter: '-LE "benchmark|perf"'
```

Comment:
This is label _exclude_, I think we want `-L` here (and `-LE` on the `NoSGX Debug` job?).

Reply:
Actually I guess we don't want the perf tests during Daily, unless we're aiming to run this job in the perf pool?

Reply:
No we don't, that's why I excluded them?

The perf stuff is run on the Perf pool by the once a day CI already, so I suggest it should stay out of the Daily altogether.

I'm less sure about the NoSGX Debug job, where we have SAN. That seems like it might usefully want to run the perf stuff, not for the numbers, but in case it's triggering safety issues that aren't otherwise visible in our slower paced integration tests.

So I'm going to go with "this is what we want here"?

Reply:
I'm convinced